client/packages/coldchain/src/Monitoring/api/TemperatureNotification/hooks/document/useTemperatureNotifications.ts

```diff
@@ -6,14 +6,24 @@ import {
 } from '@openmsupply-client/common';
 import { ListParams } from '../../api';
 
+const POLLING_INTERVAL_IN_MINUTES = 5;
+
 export const useTemperatureNotifications = (queryParams: ListParams) => {
   const api = useTemperatureNotificationApi();
   const { warning } = useNotification();
   const t = useTranslation('coldchain');
 
-  return useQuery(api.keys.paramList(queryParams), () =>
-    api.get
-      .list(queryParams)()
-      .catch(e => warning(`${t('error.fetch-notifications')}: ${e.message}`)())
+  return useQuery(
+    api.keys.paramList(queryParams),
+    () =>
+      api.get
+        .list(queryParams)()
+        .catch(e =>
+          warning(`${t('error.fetch-notifications')}: ${e.message}`)()
+        ),
+    {
+      cacheTime: 0,
```

Comment:
Did you set it 0 because the cache is not needed/not used? For example, the user never triggers this query manually because it is only run in the background on the main page? So the cache will never be used because the query is only done because of the refetchInterval?

Otherwise, it might be ok to have `refetchInterval == cacheTime` and keep `staleTime =0`. Then a user interaction would immediately get the cached query and because of staleTime=0 will query the current notifications...

Reply:
I was in two minds about that! 
The cache time 0 will disable the cache - my thinking is that we only ever want the latest info for the notification area; I rather like having it cached though, because it's an expensive query to run.

so if we set `refetchInterval == cacheTime` then the original problem will be fixed with the refetch, and the server impact will be minimised for the case when the user navigates or refreshes prior to the cache expiring. but it does mean that the user won't get the latest updates until the cache times out.. given that it's only a few minutes.. probably worth it?

Reply:
There is also staleTime which defaults to 0. This means if the user triggers the query the old data is immediately displayed when the old data is within the cacheTime but because staleTime is 0 it will also start a new query. 

Hope this makes sense.
So you could tweak staleTime and cacheTime...